kokoro/presubmit/presubmit.sh

```diff
@@ -102,12 +102,21 @@ function run_gofmt() {
   find . -name "*.go" | xargs $GOFMT -w
 }
 
+function run_pylint() {
+  python3 -m pip install pylint --user
```

Comment:
The install should not happen as part of executing this script. Developers should be able to run this script in their environment, without it changing the environment. To make the script work on the build machines, add the install to the `build.sh` script instead.